tutorials/3d/procedural_geometry/index.rst

```diff
@@ -110,19 +111,19 @@ ImmediateGeometry
 
 ImmediateGeometry is a node that uses an immediate mode style interface (like SurfaceTool) to draw objects. The
 difference between ImmediateGeometry and the SurfaceTool is that ImmediateGeometry is a node itself that can be
-added to the scene tree and is drawn directly from the code. The SurfaceTool generates a Mesh that needs to be added
```

Comment:
This whole section needs to be replaced as ImmediateGeometry was removed and replaced with [ImmediateMesh](https://docs.godotengine.org/en/latest/classes/class_immediatemesh.html)

The performance tradeoff is not so severe anymore.

Reply:
Sadly, I only know as much as what the old docs say - if you or someone else gives me bullet points of the new state of affairs I'm happy to (try to) write it up into something cohesive, but it probably makes more sense for someone who knows what's going on to do the rewrite

Reply:
Fair enough! Let's leave it as you have written it and someone else can update ImmediateGeometry to ImmediateMesh in another PR